### DISCUSSION TITLE
We found an error during import

### DISCUSSION THREAD

**Discussion by defabdullah** [community/community#134853] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I try to import private repository to my organization i get error.

We found an error during import
We recommend restarting the import process, but if you are still having trouble contact support.
A critical error occured in the Clone Source step of the git migration. Failed to clone git source

**Comment by ebndev** [comment]
Hi @defabdullah, thanks for being a part of the GitHub Community!

If you've already restarted the import process, you can contact Support by opening a ticket in the [Support Portal](https://support.github.com).

Best of luck!
